ui/src/api/index.ts

```diff
@@ -215,7 +215,7 @@ class ProjectsClient {
     const data = {
       '@type': URI.TYPE_VALUE_ATTRIBUTE,
       [URI.PROP_PREDICATE]: attributeData.property,
-      [URI.PROP_COLUMN]: attributeData.columnId,
+      [URI.PROP_COLUMN]: namedNode(attributeData.columnId),
       [URI.PROP_DATATYPE]: attributeData.dataTypeId,
```

Comment:
dataType is also a named node

Reply:
Yeah I didn't want to touch it because it will conflict with #183. Will fix once this is merged.